web/src/components/organisms/Project/Schema/hooks.ts

```diff
@@ -108,6 +108,15 @@ export default ({ projectId, modelId }: Params) => {
     refetchQueries: ["GetModels"],
   });
 
+  const handleFieldKeyUnique = useCallback(
```

Comment:
Does the backend not check if the key is unique? I feel like having the backend check and pass an error is most reliable since the model.schema.fields could become old.

Reply:
@KaWaite , this was requested by @HideBa and @yk-eukarya 
Backend does not have a mutation for this and they asked me to make it a front-end check

Reply:
Ok!